### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by davidgraeff**
Add some more build and develop info to the readme

Signed-off-by: David Gräff <david.graeff@web.de>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,14 +17,84 @@ An incomplete list of other repositories follows below:
...
+|
++-- CODEOWNERS  This file assigns people to directories so that they are informed if a pull-request
+                would modify that directory/binding.
+```
+
+### Command line build
+
 To build all add-ons from the command-line, type in:
-`mvn clean install`
+`mvn clean install -DskipChecks -DskipTests`. This will only build all add-ons but does not run the tests and the static code checks. For binding development you want to run that command without skipping checks and tests.
```
Comment (martinvw) [inline README.md]: I would not want to promote skip tests and checks so actively...

Comment (davidgraeff) [reply README.md]: If you just want to build jar files (remember that this repo is existing as forks on other developers git repositories) skipping tests and checks is reasonable I think (7 minutes vs 30 minutes). But yeah can be more emphasized that it is not a good idea to develop yourself without those helpers. I'm happy for suggestions.

File: README.md [hunk 2]
```diff
@@ -17,14 +17,84 @@ An incomplete list of other repositories follows below:
...
+
+Subsequent calls can include the `-o` for offline as in: `mvn clean install -DskipChecks -o` which will be a bit faster.
+
+For integration tests you might need to run: `mvn clean install -DwithResolver -DskipChecks`.
+
+### How to develop in the Eclipse IDE
+
+1. Install Bndtools in your Eclipse IDE. (Is automatically installed if you have used the Eclipse Installer for openHAB development as described here: https://www.openhab.org/docs/developer/development/ide.html)
+1. Checkout the bnd based openHAB demo application: `git clone --depth=1 https://github.com/maggu2810/openhab-demo`.
```
Comment (cweitkamp) [inline README.md]: Can you fix the numeration?

File: README.md (hunk 1 above)
Comment (martinvw) [reply README.md]: To build all add-ons from the command-line, type in:

`mvn clean install`

Optionally you can skip tests (`-DskipTests`) or skip some static analysis (`-DskipChecks`) this does  improve the build time but could hide problems in your code. For binding development you want to run that command without skipping checks and tests.

**Comment by davidgraeff** [comment]
Incorporated

**Comment by martinvw** [comment]
Thanks @davidgraeff

**Comment by J-N-K** [comment]
Nice. Maybe you could add that you can build a single bundle from
`poms/bnd` with `mvn clean install -pl ../../bundles/<name>`. I find that very useful.

**Pull request merged**